instrumentation/log4j/log4j-appender-2.17/library/src/test/java/io/opentelemetry/instrumentation/log4j/appender/v2_17/internal/LogEventMapperTest.java

```diff
@@ -153,7 +158,8 @@ void testCaptureMapMessageWithoutSpecialAttribute() {
   void testCaptureStructuredDataMessage() {
     // given
     LogEventMapper<Map<String, String>> mapper =
-        new LogEventMapper<>(ContextDataAccessorImpl.INSTANCE, false, true, singletonList("*"));
+        new LogEventMapper<>(
+            ContextDataAccessorImpl.INSTANCE, false, true, false, singletonList("*"));
```

Comment:
it may be worth introducing `LogEventMapperBuilder` in the future

Reply:
Agree, same thing for Logback instrumentation.